prdoc/pr_1913.prdoc

```diff
@@ -0,0 +1,18 @@
+title: BEEFY on Polkadot
+
+doc:
+  - audience: Core Dev
+    description: |
+      The BEEFY gadget has been enabled on Polkadot by default.
+      If experiencing stability issues caused by BEEFY, it can be disabled using `--no-beefy` flag.
```

Comment:
```suggestion
      The BEEFY gadget has been enabled on Polkadot by default.
      If experiencing stability issues caused by BEEFY, it can be disabled using `--no-beefy` flag.
      BEEFY doesn't (yet) support warp sync. So, attempting to Warp sync as a validator will throw an error.
```